spark/src/main/res/values/strings.xml

```diff
@@ -30,4 +30,7 @@
     <string name="spark_rating_a11y">Note de %1$.1f</string>
 
     <string name="spark_user_avatar_content_description">Photo de profil</string>
+
+    <string name="spark_annotatedStringResource_test"><b>Best</b> <annotation typography="large">practices</annotation> for text on <annotation color="primary">Android</annotation></string>
```

Comment:
nit: this should go into a test source. We don't want to ship this in the aar.

Reply:
I'll remove it once we can render all these styles in the sample app as the layout lib doesn't support rendering of annotated string yet